cypress/support/index.js

```diff
@@ -1,3 +1,3 @@
 Cypress.Screenshot.defaults({
-  screenshotOnRunFailure: false,
+  screenshotOnRunFailure: true,
```

Comment:
I have set screenshots to be turned on when tests fail - this is to enable us to debug issues on Live.